### ISSUE TITLE
Feature Request: Add UV & Pollution Data

### ISSUE THREAD

**Issue by ztroop** [billykwooten/openweather-exporter#19]
Great project! Thank you for sharing this. I was looking over `openweathermap` module and was wondering if we could add UV and Pollution data to the `openweather-exporter` collection.

That would mean pulling from `owm.NewUV` and `owm.NewPollution` respectively. If that's not something you find useful for this project, I can implement the logic on my own. I know UV data is available in "One Call", but pollution would be a separate call. Thoughts?

**Comment by billykwooten** [comment]
I'll be working on this soon! Apologies for the slow response. If you happen to have a PR though I can definitely take a look and approve it as well.

**Comment by ztroop** [comment]
I started experimenting with the `openweathermap`, but realized it's pretty out of date. I decided just to make my own `owm` module to provide `OneCall` and `Pollution` functionality to drop the dependency. Given that I made a large number of changes, I don't think my PR would be too helpful. Feel free to take a [look at my fork](https://github.com/ztroop/openweather-exporter) if you want to pull in some of those changes. 

I tackled a few other things:

- Added `snow1h`, as it was missing from `Describe`.
- Opinionated, maybe questionable, choices like removing `currentconditions`, as I didn't see much value in that. I also removed comments because they didn't tell me anything I don't already know from already reading the code.
- In addition to UVI and AQI, I've also added individual air concentrations types.
- Changing defaults for the sake of my own fork usage.
